src/modules/services/submodule/mainnetSwap/submodule2/bsc/views/ViewServicesMainnetSwapBSCToSirius.vue

```diff
@@ -235,6 +236,11 @@ export default {
   setup() {
     let verifyingTxn;
 
+    const verifyMetaMaskPlugin = ref(true);
```

Comment:
should use `isMetaMaskPlugin` instead of `verifyMetaMaskPlugin`, as verify sound more like action than boolean variable. Just saying